pinot-segment-local/src/test/java/org/apache/pinot/segment/local/utils/TableConfigUtilsTest.java

```diff
@@ -1755,6 +1755,14 @@ public void testTaskConfig() {
     } catch (IllegalStateException e) {
       Assert.assertTrue(e.getMessage().contains("has invalid aggregate type"));
     }
+
+    // valid agg
+    HashMap<String, String> validAggConfig = new HashMap<>(realtimeToOfflineTaskConfig);
```

Comment:
could we also add a test for invalid agg please?

Reply:
it was already there, on the lines above

Reply:
oh so we dont differentiate invalid agg vs valid agg with invalid configured agg?

Reply:
anything available in the `ValueAggregatorFactory` methods should be valid here, which is why I've implemented it this way

Reply:
I've added a test case above for a valid `AggregationFunctionType` which has no `ValueAggregator` available in the `ValueAggregatorFactory`, I picked `histogram` for this because that doesn't have one right now, and I don't think would make sense to have one

Reply:
Ah. I see. Ok that sounds good